src/notation/internal/notationaccessibility.cpp

```diff
@@ -165,15 +165,20 @@ QString NotationAccessibility::rangeAccessibilityInfo() const
         endSegment = endSegment->prev1MM();
     }
 
-    std::pair<int, float> startBarbeat = selection()->startSegment()->barbeat();
-    QString start =  muse::qtrc("notation", "Start measure: %1; Start beat: %2")
-                    .arg(startBarbeat.first)
-                    .arg(startBarbeat.second);
-
-    std::pair<int, float> endBarbeat = endSegment->barbeat();
-    QString end =  muse::qtrc("notation", "End measure: %1; End beat: %2")
-                  .arg(endBarbeat.first)
-                  .arg(endBarbeat.second);
+    EngravingItem::BarBeat startBarbeat = selection()->startSegment()->barbeat();
+
+    QString start = muse::qtrc("engraving", "Start measure: %1").arg(String::number(startBarbeat.bar));
+    if (startBarbeat.displayedBar != startBarbeat.bar) {
+        start += muse::qtrc("engraving", "; Start displayed measure: %1").arg(startBarbeat.displayedBar);
```

Comment:
Preferably take the `"; "` outside the `qtrc`, like you do in `spanner.cpp`, so that translators don't need to worry about that and can't make mistakes with that.